<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1563</width>
    <height>962</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_artist">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>129</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>歌手 </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_aumble">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>911</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>911</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_position">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑  </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_duration">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>专辑  </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1563</width>
     <height>48</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
